src/server/server.ts

```diff
@@ -131,11 +135,25 @@ async function serveArticle(
 		throw new Error('Could not get image salt');
 	}
 
+	let inlineJS = '';
+	if (isEditions) {
+		const hashResponse = await fetch(
+			`https://mobile.code.dev-guardianapis.com/assets/hashed-names.json`,
```

Comment:
Do we need to update this to use `mobile.guardianapis.com` ?

Reply:
Yep, updated!